test/alternative_spec.rb

```diff
@@ -0,0 +1,40 @@
+require_relative "spec_helper.rb"
+require "update-alternatives/model/alternative"
+
+describe UpdateAlternatives::Alternative do
+
+  describe ".all" do
+    subject { UpdateAlternatives::Alternative.all }
+
+    it "returns an array of Alternative objects" do
+      alternatives_pip_with_two_choices_stub
+      expect(subject.class).to eq Array
+      subject.all? { |e| expect(e).to be_a(UpdateAlternatives::Alternative) }
```

Comment:
Using `#all?` makes no much sense here. The result is not relevant. You actually wanted to do this
```ruby
expect(subject).to all(be_a(UpdateAlternatives::Alternative))
```